script.js

```diff
@@ -0,0 +1,318 @@
+/* eslint-disable no-plusplus */
+
+const divWrapper = document.createElement('div');
+divWrapper.className = 'wrapper';
+document.body.append(divWrapper);
+
+const textArea = document.createElement('textarea');
+textArea.className = 'textarea';
+divWrapper.append(textArea);
+
+const divKeyboard = document.createElement('div');
+divKeyboard.className = 'keyboard';
+divWrapper.append(divKeyboard);
+
+const divRow1 = document.createElement('div');
+divRow1.className = 'row1';
+divKeyboard.append(divRow1);
+
+const divRow2 = document.createElement('div');
+divRow2.className = 'row2';
+divKeyboard.append(divRow2);
+
+const divRow3 = document.createElement('div');
+divRow3.className = 'row3';
+divKeyboard.append(divRow3);
+
+const divRow4 = document.createElement('div');
+divRow4.className = 'row4';
+divKeyboard.append(divRow4);
+
+const divRow5 = document.createElement('div');
+divRow5.className = 'row5';
+divKeyboard.append(divRow5);
+
+const pDescriptionText = document.createElement('p');
```

Comment:
Bad variable name. I recommend to read this article: https://learn.javascript.ru/ninja-code